sector-builder-ffi/src/api.rs

```diff
@@ -542,6 +546,120 @@ pub unsafe extern "C" fn sector_builder_ffi_read_piece_from_sealed_sector(
     })
 }
 
+/// Imports a sector sealed elsewhere and takes ownership of its metadata,
+/// cache directory, and sealed sector file.
+///
+#[no_mangle]
+pub unsafe extern "C" fn sector_builder_ffi_import_sealed_sector(
+    ptr: *mut SectorBuilder,
+    sector_id: u64,
+    sector_cache_dir_path: *const libc::c_char,
+    sealed_sector_path: *const libc::c_char,
+    seal_ticket: FFISealTicket,
+    seal_seed: FFISealSeed,
+    comm_r: &[u8; 32],
+    comm_d: &[u8; 32],
+    p_aux_comm_c: &[u8; 32],
+    p_aux_comm_r_last: &[u8; 32],
+    proof_ptr: *mut u8,
+    proof_len: libc::size_t,
+    pieces_ptr: *mut FFIPieceMetadata,
+    pieces_len: libc::size_t,
+) -> *mut types::ImportSealedSectorResponse {
+    catch_panic_response(|| {
+        init_log();
+
+        info!("import_sealed_sector: {}", "start");
+
+        let mut response: types::ImportSealedSectorResponse = Default::default();
+
+        let comm_r_last = PedersenDomain::try_from_bytes(p_aux_comm_r_last);
+        let comm_c = PedersenDomain::try_from_bytes(p_aux_comm_c);
+
+        if comm_r_last.is_err() {
+            response.status_code = FCPResponseStatus::FCPUnclassifiedError;
+            response.error_msg = rust_str_to_c_str("cannot xform comm_r_last to PedersenDomain");
```

Comment:
what is xform?

Reply:
shorthand for "transform" - just updated it to say "convert"